src/render/TerraformRenderer.js

```diff
@@ -50,23 +51,40 @@ class TerraformRenderer extends DefaultRender {
         },
         new Map(),
       ),
+      parentEventId,
     );
   }
 
   /**
    * Render files from related components.
    *
    * @param {Map<string,Component>} map - Component mapped by file name.
+   * @param {string} [parentEventId=null] - Parent event id.
```

Comment:
same

Reply:
It's manage by linter and jsdoc, if you set String we have a linter error. So it's good like that !